translations/kubectl/pt_BR/LC_MESSAGES/k8s.po

```diff
@@ -0,0 +1,3319 @@
+"image, and switching the\n"
+"\t\t# name of the replication controller.\n"
+"\t\tkubectl rolling-update frontend-v1 frontend-v2 --image=image:v2\n"
+"\n"
+"\t\t# Update the pods of frontend by just changing the image, and keeping "
+"the old name.\n"
+"\t\tkubectl rolling-update frontend --image=image:v2\n"
+"\n"
+"\t\t# Abort and reverse an existing rollout in progress (from frontend-v1 to "
+"frontend-v2).\n"
+"\t\tkubectl rolling-update frontend-v1 frontend-v2 --rollback"
+msgstr ""
+"\n"
+"\t\t# Atualiza os pods de frontend-v1 utilizando os dados do novo "
+"replication controller definido em frontend-v2.json.\n"
+"\t\tkubectl rolling-update frontend-v1 -f frontend-v2.json\n"
+"\n"
+"\t\t# Atualiza os pods do frontend-v1 utilizando os dados em JSON passados "
+"pelo stdin.\n"
+"\t\tcat frontend-v2.json | kubectl rolling-update frontend-v1 -f -\n"
+"\n"
+"\t\t# Atualiza os pods do frontend-v1 para frontend-v2 trocando a imagem, e "
+"trocando o\n"
+"\t\t# nome do replication controller.\n"
+"\t\tkubectl rolling-update frontend-v1 frontend-v2 --image=image:v2\n"
+"\n"
+"\t\t# Atualiza os pods do frontend trocando a imagem, e mantendo o nome "
+"antigo.\n"
+"\t\tkubectl rolling-update frontend --image=image:v2\n"
+"\n"
+"\t\t# Cancela e reverte um rollout existente em progresso (de frontend-v1 "
+"para frontend-v2).\n"
+"\t\tkubectl rolling-update frontend-v1 frontend-v2 --rollback"
+
+#: pkg/kubectl/cmd/apply_view_last_applied.go:52
+msgid ""
+"\n"
+"\t\t# View the last-applied-configuration annotations by type/name in YAML.\n"
+"\t\tkubectl apply view-last-applied deployment/nginx\n"
+"\n"
+"\t\t# View the last-applied-configuration annotations by file in JSON\n"
+"\t\tkubectl apply view-last-applied -f deploy.yaml -o json"
+msgstr ""
+"\n"
+"\t\t# Visualiza a anotação last-applied-configuration annotations pelo tipo/"
+"nome no YAML.\n"
+"\t\tkubectl apply view-last-applied deployment/nginx\n"
+"\n"
+"\t\t# Visualiza a anotação last-applied-configuration no arquivo JSON\n"
+"\t\tkubectl apply view-last-applied -f deploy.yaml -o json"
+
+#: pkg/kubectl/cmd/apply.go:75
+msgid ""
+"\n"
+"\t\tApply a configuration to a resource by filename or stdin.\n"
+"\t\tThis resource will be created if it doesn't exist yet.\n"
+"\t\tTo use 'apply', always create the resource initially with either 'apply' "
+"or 'create --save-config'.\n"
+"\n"
+"\t\tJSON and YAML formats are accepted.\n"
+"\n"
+"\t\tAlpha Disclaimer: the --prune functionality is not yet complete. Do not "
+"use unless you are aware of what the current state is. See https://issues."
+"k8s.io/34274."
+msgstr ""
+"\n"
+"\t\tAplica a configuração em um recurso usando um nome de arquivo ou stdin.\n"
+"\t\tEste recurso será criado se ele não existir.\n"
+"\t\tPara utilizar o 'apply', sempre crie o recurso inicialmente com 'apply' "
+"ou 'create --save-config'.\n"
+"\n"
+"\t\tFormatos JSON e YAML são aceitos.\n"
+"\n"
+"\t\tNota Alpha: a funcionalidate --prune não está completa. Não utilize a "
```

Comment:
typo in funcionalidade here

Reply:
maybe need to update the main strings. will do a followup on that, since this is from the main english one